### PR TITLE
document managment inclusion

### PR DESCRIPTION
**Pull request by alectronic0**
Evidence / Document Management has been added 
Microservice Included:
- DM App
-DM API GW Web
-DM Viewer

screenshot from 2018-01-10 10-38-20

### CODE REVIEW

File: docs/microservices.json [hunk 1]
```diff
@@ -27,6 +27,10 @@
         {
             "name": "CMC",
             "colour": "#f30000"
+        },
+        {
+            "name": "Evidence",
+            "colour": "#fff200"
```
Comment (rkondratowicz) [inline docs/microservices.json]: Hard to see this yellow on white background imho. Perhaps brown could be used instead? 
We're running out of colors 😨

Comment (doncem) [reply docs/microservices.json]: that's why i had 10 initial pastel colours init.

**Comment by jasonpaige** [comment]
Does Document Management viewer belong here? I'm just thinking that if it's a component which a team installs like a library to use within an app, then we'll have several instances of it running and I don't think we would want that node repeated for each team.
I may have misunderstood what the DM Viewer is though, so sorry if I've totally missed the point there!
What do you reckon?

**Comment by alectronic0** [comment]
@jasonpaige this is the viewer that will be the basis of annotation, redactions, etc. so it will be its own service with its own backend & frontend (atm this is purely frontend)

File: docs/microservices.json (hunk 1 above)
Comment (alectronic0) [reply docs/microservices.json]: will change the colour now

Comment (alectronic0) [reply docs/microservices.json]: screenshot from 2018-01-10 11-50-06
How about 'http://www.color-hex.com/color/8b4513' http://www.color-hex.com/color/8b4513

**Pull request merged**